### PR TITLE
#51 Creates Custom Tags taxonomies

### LINKED ISSUE #51: Add Open Calais taxonomy volcabs
- opencalais_city_tags
- opencalais_continent_tags
- opencalais_country_tags
- opencalais_provinceorstate_tags
- opencalais_region_tags
- opencalais_person_tags
- opencalais_socialtags_tags

### PR DESCRIPTION
**Pull request by brandonhundt**
Closes #51 

- Adds City, Continents, Countries, Person, Province or State, Region, and Social tags to posts --- all managed in the new tw_custom_tags plugin

## To Review

- [ ] Checkout Branch.
- [ ] Run `npm run local` (or `npm run refresh` if you want a fresh database.)
- [ ] Go to http://the-world-wp.lndo.site/.

> ...then...

- [ ] Create/edit a post
- [ ] add terms to each to each of the custom taxonomies.
- [ ] Publish the post
- [ ] Confirm the values show up in Rest

### CODE REVIEW

**Review by rpeterman-gp — CHANGES_REQUESTED** [review]
Can you resave the new plugin file and address the linter requirements for the docblock comments?

**Comment by brandonhundt** [comment]
@rpeterman-gp Fixed!

**Review by rpeterman-gp — CHANGES_REQUESTED** [review]
Can we break tw-custom-tags file up into a file per taxonomy, then use a `require_once` to bring them into the `tw-custom-tags.php` file? Would rather deal with many files with a specific focus, than one super long file.

File: wp-content/plugins/tw-custom-tags/tw-custom-tags.php [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?php
+/*
+Plugin Name: TW Custom Tags
+Description: Creates the custom taxonomies for metadata
+*/
+
+require_once('taxonomies/taxonomy-city.php');
```
Comment (brandonhundt) [inline wp-content/plugins/tw-custom-tags/tw-custom-tags.php:7]: @rpeterman-gp I've changed this file to break it out into separate files. Let me know if there are further enhancements I can make.

**Pull request merged**